05 Lean CLI/03 Tutorials/06 Local data/02 Generating random data/02 Running the generator.html

```diff
@@ -10,6 +10,7 @@
 <pre>$ lean data generate --start 20150101 --symbol-count 10
 Begin data generation of 10 randomly generated Equity assets...</pre>
 </div>
-        You can also specify an end date using <code>--end &lt;yyyyMMdd&gt;</code>, generate data for a different security type using <code>--security-type &lt;type&gt;</code>, for a different resolution using <code>--resolution &lt;resolution&gt;</code>, or with a different density using <code>--density &lt;density&gt;</code>.
+        You can also specify an end date using <code>--end &lt;yyyyMMdd&gt;</code>, generate data for a different security type using <code>--security-type &lt;type&gt;</code>, for a different resolution using <code>--resolution &lt;resolution&gt;</code>, or with a different density using <code>--data-density &lt;density&gt;</code>.
     </li>
+    <li>Run <code>lean data generate --help</code> for the full list of options</li>
```

Comment:
Suggest moving this sentence to the end of the previous list item. The list explains how to generate random data, running `lean data generate --help` is not a required step for that.